recipe/build.sh

```diff
@@ -31,7 +31,9 @@ bash configure --prefix=$PREFIX \
                --with-geotiff=$PREFIX \
                --with-hdf4=$PREFIX \
                --with-hdf5=$PREFIX \
-               --with-jpeg=$PREFIX \
+               --with-jpeg=internal \
+               --with-jpeg12 \
+               --with-hide-internal-symbols=yes \
```

Comment:
I *think* this will hide jpeg symbols so there won't be clashes with `libjpeg`. Hopefully this doesn't cause other problems....